src/ansiblelint/__main__.py

```diff
@@ -32,11 +32,15 @@
 
 from enrich.console import should_do_markup
 
+# pylint: disable=unused-import
+import ansiblelint._preimport  # must be kept before other ansiblelint imports!
```

Comment:
This is a dirty hack. Instead, expose a function and call it at the beginning of the runtime.

Reply:
I refactored it to load from config, at the same location where we do something similar, as that command does call ansible.